uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/loyalty/LoyaltyPresenter.kt

```diff
@@ -148,6 +152,7 @@ class LoyaltyPresenter(val userStore: UserStore = KarhooApi.userStore,
                         userStore.loyaltyStatus = result.data
                         set(result.data)
 
+                        updateBalancePoints()
```

Comment:
Can we update the Unit tests? 
For the case that the points are `null` and the case there is valid points